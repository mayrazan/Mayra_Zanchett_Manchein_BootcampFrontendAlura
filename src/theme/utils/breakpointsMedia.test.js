import { css } from 'styled-components';
import breakpointsMedia from './breakpointsMedia';

describe('breakpointsMedia()', () => {
  describe('when receives a breakpoint', () => {
    test('and its an object', () => {
      const media = {
        md: css`
          text-align: center;
        `,
      };
      const result = breakpointsMedia(media);

      expect(result).toMatchSnapshot();
    });
  });
});
