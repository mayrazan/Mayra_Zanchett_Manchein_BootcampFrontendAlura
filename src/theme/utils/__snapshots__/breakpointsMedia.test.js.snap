// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`breakpointsMedia() when receives a breakpoint and its an object 1`] = `
Array [
  Array [
    "@media screen and (min-width:",
    "768",
    "px){",
    "text-align:center;",
    "}",
  ],
]
`;
